packages/ckeditor5-dev-env/tests/translations/upload.js

```diff
@@ -10,73 +10,91 @@
 const path = require( 'path' );
 const sinon = require( 'sinon' );
 const mockery = require( 'mockery' );
-const transifexService = require( '../../lib/translations/transifex-service' );
-const fs = require( 'fs' );
 
 describe( 'upload', () => {
-	let sandbox;
-	let upload;
+	let sandbox, stubs, upload;
+	let packageNames, serverResources, fileContents;
```

Comment:
I don't like merging variables into a one big line ;)
In this case these 2 lines differs. In the first there're declared variables that are initialised in the `beforeEach()` hook, while in the second there're declared variables for the later initialisations in the tests.